app/lib/methods/migrateMMKVStorage.ts

```diff
@@ -0,0 +1,67 @@
+import { MMKV } from 'react-native-mmkv';
+
+import MMKVReader from '../native/NativeMMKVReader';
+import { isAndroid } from './helpers';
+import userPreferences from './userPreferences';
+
+export async function migrateFromOldMMKV(oldInstanceId: string = 'default', newStorage?: MMKV) {
+	const errors: string[] = [];
+	userPreferences.setBool('WORKSPACE_MIGRATION_COMPLETED', false);
```

Comment:
What's the difference between `WORKSPACE_MIGRATION_COMPLETED` and `MMKV_MIGRATION_COMPLETED`?